packages/types/tsconfig.json

```diff
@@ -9,8 +9,9 @@
         "esModuleInterop": true,
         "allowSyntheticDefaultImports": true,
         "jsx": "react",
-        "outDir": "./lib"
+        "outDir": "./lib",
+        "rootDir": "./src",
+        "composite": true
```

Comment:
`composite` is needed to use project references. It ensures that some other important flags are set, and I think it's what generates the `tsbuildinfo` files